RELEASING.md

```diff
@@ -0,0 +1,99 @@
+# Releasing Sourcery
+
+There're no hard rules about when to release Sourcery. Release bug fixes frequently, features not so frequently and breaking API changes rarely.
+
+### Release
+
+Example is for releasing `0.6.1` version of the Sourcery. Make sure you've been added as owner for CocoaPods and have set up the api token to be able to upload releases on GitHub.
```

Comment:
I've added couple of links, that looks fine to me but we can add more details if you think it helps. Check b294b9e